sdks/python/apache_beam/options/pipeline_options_validator_test.py

```diff
@@ -370,6 +372,50 @@ def test_validate_dataflow_job_file(self):
     errors = validator.validate()
     self.assertFalse(errors)
 
+  def test_worker_num_is_positive(self):
```

Comment:
Same here (and for other test functions).
```suggestion
  def test_num_workers_is_positive(self):
```